.gitignore

```diff
@@ -0,0 +1,9 @@
+/.idea/.
+/.mvn/.
+lib/.
+target//
+mvnw
+mvnw.cmd
+Storehouse.iml
+.DS_Store
+spring-boot-flyway-demo.iml
```

Comment:
Folders idea, lib, target  and file Storehouse.iml should not be present in pull request

Reply:
it's gitignore issue :(